.github/workflows/issue-manager.yml

```diff
@@ -0,0 +1,30 @@
+name: Issue Manager
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  issue_comment:
+    types:
+      - created
+  issues:
+    types:
+      - labeled
+  pull_request_target:
+    types:
+      - labeled
+  workflow_dispatch:
+
+jobs:
+  issue-manager:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: tiangolo/issue-manager@0.5.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          config: >
+            {
+              "info-needed": {
+                "delay": 1209600,
```

Comment:
Might be worth adding a comment here specifying that this is 14 days =P

Reply:
Good point. Unfortunately the config is raw JSON and JSON does not really support comments. However, delay also accepts ISO 8601 according to the [issue manager docs](https://github.com/tiangolo/issue-manager/tree/0.5.0?tab=readme-ov-file#delay). So I used that instead. Now it reads `P14D` which means it's a period of 14 days 😊